src/modules/editor/SoundEditor/Note.ts

```diff
@@ -0,0 +1,57 @@
+
+import * as Tone from "tone";
+import { SampleLibrary } from "@modules/editor/SoundEditor/Tonejs-Instruments"
+
+
+
+class Note {
+  public samp: Tone.Sampler;
+  constructor(
+    public note: string = "Nan",
+    private _duration: number = 1,
+    public instrument: string = "piano",
+  ) {
+    this.note = note;
+    this._duration = _duration;
+    this.instrument = instrument;
+    if (this.note == "Nan") {
+      this.samp = new Tone.Sampler();
+    } else {
+      this.samp = SampleLibrary.load({
+        instruments: this.instrument,
+        /* onload: () => {
```

Comment:
Please remove those comments if you don't think that you're going to use those snippets anything soon :]